pkg/repository/v1/output.go

```diff
@@ -37,10 +39,16 @@ func (e *TaskOutputEvent) IsCancelled() bool {
 	return e.EventType == sqlcv1.V1TaskEventTypeCANCELLED
 }
 
-func NewCompletedTaskOutputEventFromTask(task *sqlcv1.V1Task, output []byte) *TaskOutputEvent {
+func NewSkippedTaskOutputEventFromTask(task *sqlcv1.V1Task, output []byte) *TaskOutputEvent {
 	e := baseFromTasksRow(task)
 	e.Output = output
 	e.EventType = sqlcv1.V1TaskEventTypeCOMPLETED
+
+	if task.DesiredWorkerID.Valid {
+		workerId := sqlchelpers.UUIDToStr(task.DesiredWorkerID)
+		e.WorkerId = &workerId
+	}
+
```

Comment:
some of these changes look like they might've leaked over from the affinity PR?

Reply:
should be a little cleaner now 👍